goleveldb.go

```diff
@@ -24,7 +25,13 @@ type GoLevelDB struct {
 var _ DB = (*GoLevelDB)(nil)
 
 func NewGoLevelDB(name string, dir string) (*GoLevelDB, error) {
-	return NewGoLevelDBWithOpts(name, dir, nil)
+	options := &opt.Options{
+		Filter:                 filter.NewBloomFilter(10), // by default, goleveldb doesn't use a bloom filter.
+		OpenFilesCacheCapacity: 8192, // default value here is 200 on mac, 500 on other.
+		BlockCacheCapacity:     opt.GiB, // The default value is 8MiB.
+		WriteBuffer:            64 * opt.MiB, // The default value is 4MiB.
```

Comment:
these are hefty jumps, what is the reasoning?

Reply:
I think these are way to high and we should delete them and opt to the default then figure out how to proceed instead of throwing random big numbers out there

Reply:
I'd be okay with that.  The bloom filter should be a pretty big boost by itself.  1 moment please...

Reply:
thanks